spec/unit/installer/xcode/pods_project_generator/pod_target_integrator_spec.rb

```diff
@@ -0,0 +1,81 @@
+require File.expand_path('../../../../../spec_helper', __FILE__)
```

Comment:
Missing tests for this class now added! Test integration with test targets and shell script phases!